src/form-field/internal.tsx

```diff
@@ -91,7 +94,7 @@ export default function InternalFormField({
       )}
 
       <div className={clsx(styles.controls, __hideLabel && styles['label-hidden'])}>
-        <InternalGrid gridDefinition={gridDefinition}>
+        <InternalGrid gridDefinition={gridDefinition} disableGutters={__disableGutters}>
```

Comment:
Why is this needed? After hiding the label and disabling gutters, do you have anything else left from the form field functionality? If you need validation message, it can be rendered using `<FormFieldError>` sub component directly

Reply:
It's also needed for rendering constraint text and connecting ARIA properties.

Reply:
The point remains the same. Instead of conditionally turning off stuff we should use composition. Extract the form field basics and use them here instead